ruby_test.rb

```diff
@@ -0,0 +1,2 @@
+puts "one"
+puts 'two'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.